bundles/org.openhab.binding.mycroft/README.md

```diff
@@ -0,0 +1,123 @@
+# Mycroft Binding
+
+This binding will connect to Mycroft A.I. in order to control it or react to events by listening on the message bus.
+
+Possibilies include :
+
+- Press a button in openHAB to wake Mycroft without using a wake word.
+- Simulate a voice command to launch a skill, as if you just spoke it
+- Send some text that Mycroft will say (Using its Text To Speech service)
+- Control the music player
+- Mute or duck the sound volume of Mycroft
+- React to all the aforementioned events ...
+- ... And send/receive all other kind of messages on the message bus
+
+
+## Supported Things
+
+The only thing managed by this binding is a Mycroft instance.
```

Comment:
Can you name directly the thing type id here?

Reply:
I didn't find a more suited world to describe "a Mycroft" (as you guessed, the world should encompass the hardware version, and the picroft version, and every kind of Mycroft running on any computer ?)
Do you have something in mind, or should I just go for a brief description behind "instance" ? I think will put a description, tell me if you'd rather something else.
"The only thing managed by this binding is a Mycroft instance. (e.g. Mark I/II, Picroft, or any other variant)"